examples/Accessor.cpp

```diff
@@ -58,6 +58,12 @@ struct UserExample : tvgexam::Example
             return true;
         };
 
+        if (auto paint = picture->paint(tvg::Accessor::id("star"))) {
+            auto shape = static_cast<tvg::Shape*>(const_cast<tvg::Paint*>(paint));
+            shape->strokeFill(255, 255, 0);
+            shape->strokeWidth(5);
+        }
+
         if (!tvgexam::verify(accessor->set(picture.get(), f, nullptr))) return false;
 
```

Comment:
1. Suggestion to move your addition after 67 line to not break the context.
2. Appreciate you if you adding a comment what you're doing here for the users who really want to learn from your code something like this:
```
// Try to retrieve the shape that corresponds to the SVG node with the unique ID "star".
```

Reply:
I updated PR, Thanks!